### PR TITLE
Merge keychains

### PR DESCRIPTION
**Pull request by greg-schrammel**
Fixes BX-1602
Figma link (if any):

## What changed (plus any additional context for devs)

merge pks to it's hd group when possible
in the clip I added by pk the index 2 of a seed phrase, later when I add this seed phrase and click to create a wallet in the second index, it merges both into a single group (removes the pk keychain and merges all into the hd)

https://github.com/rainbow-me/browser-extension/assets/6232729/8433998e-5cfb-49eb-8500-9e017671532d

https://github.com/rainbow-me/browser-extension/assets/6232729/fadea7c2-c088-46da-a6fc-9a62a4a594b0

importing a duplicate pk, will just select the account

https://github.com/rainbow-me/browser-extension/assets/6232729/068ffb7e-c07a-486e-bdde-b9eec0325713

trying to import the same seed twice with prompt an error with "This seed is imported already" (maybe we need a better message here)

### CODE REVIEW

**Review by estebanmino — COMMENTED** [review]
@greg-schrammel could you explain what you did here?

File: src/entries/popup/components/ImportWallet/ImportWalletSelection.tsx [hunk 1]
```diff
@@ -69,17 +70,14 @@ const useDeriveAccountsFromSecrets = (secrets: string[]) => {
...
-    accountsIgnored = [],
-  }: {
-    secrets: string[];
-    accountsIgnored?: Address[];
-  }) => {
-    setIsImporting(true);
-    return (async () => {
+  const { mutateAsync: importSecrets, isLoading: isImporting } = useMutation({
+    mutationFn: async ({
+      secrets,
```
Comment (estebanmino) [inline src/entries/popup/components/ImportWallet/ImportWalletSelection.tsx:75]: does this use cache to store results? if so maybe is not a good idea to use secrets for it

**Review by estebanmino — COMMENTED** [review]
we need some qa for this

**Review by estebanmino — APPROVED** [review]
seems like i mentally approved this pr last week but didn't press the ✅  button

**Review by derHowie — APPROVED** [review]

**Comment by DanielSinclair** [comment]
@greg-schrammel After you wrap up Degen Mode, if you could have a look at this and merge

**Pull request merged**